rosserial_client/src/rosserial_client/make_library.py

```diff
@@ -41,7 +41,7 @@
 import roslib.srvs
 import roslib.message
 import traceback
-
+import rospkg
```

Comment:
Is it better just to remove 'import rospkg'? It was needed by the block of code that I've deleted later. Right now there are no direct usages of 'rospkg'.